core/src/avm2/object.rs

```diff
@@ -245,72 +241,6 @@ pub trait TObject<'gc>: 'gc + Collect + Debug + Into<Object<'gc>> + Clone + Copy
         self.get_property_local(&name, activation)
     }
 
-    /// Retrieve a property by Multiname lookup.
-    ///
-    /// This method should not be overridden.
-    ///
-    /// This corresponds directly to the AVM2 operation `getproperty`, with the
-    /// exception that it does not special-case object lookups on dictionary
-    /// structured objects.
-    #[allow(unused_mut)] //Not unused.
-    #[no_dynamic]
-    fn get_property(
```

Comment:
why not keep these, if only as
```
    fn get_property(...) {
        Value::from(self).get_property(...)
    }
```
?
This would prevent several dozen `Value::from(self).get_property(...)` changes in other places, and feels like a nice QoL wrapper to have in general.